src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -333,6 +342,31 @@ private RouteFW wrapRoute(
         return routeRO.wrap(buffer, index, index + length);
     }
 
+    private long resolveChallengeResponseTimeout(
+        JsonWebSignature verified)
+    {
+        long challengeResponseTimeout = 0;
+
+        try
+        {
+            if (verified != null)
+            {
+                final JwtClaims claims = JwtClaims.parse(verified.getPayload());
+                final NumericDate challengeAfterDate = claims.getNumericDateClaimValue(
+                        config.geClaimNamespace() + config.getClaimNameChallengeResponseTimeout());
```

Comment:
Use `String.format("%s%s", config.geClaimNamespace(), config.getClaimNameChallengeResponseTimeout())` in `OAuthProxyFactory` constructor to compute `challengeResponseTimeoutClaimName` once.